pages/clever-cloud.md

```diff
@@ -0,0 +1,172 @@
+Login successful as ...
+</pre>
+
+
+## Create your Clever Cloud application
+
+1. If you are using maven `clever create --type maven [your application name]` or using gradle `clever create --type gradle [your application name]`
+
+2. Add a database addon to your application `clever addon create [addon provider] [your addon name] --link [your application name]`
+
+    List supported addon providers `clever addon providers`
+    <pre>
+    cellar-addon      Cellar S3 storage       S3-like online file storage web service
+    config-provider   Configuration provider  Expose configuration to your applications  (via environment variables)
+    es-addon          Elastic Stack           Elasticsearch with Kibana and APM server as options
+    fs-bucket         FS Buckets              Persistent file system for your application
+    mongodb-addon     MongoDB                 A noSQL document-oriented database
+    mysql-addon       MySQL                   An open source relational database management system
+    postgresql-addon  PostgreSQL              A powerful, open source object-relational database system
+    redis-addon       Redis                   Redis by Clever Cloud is an in-memory key-value data store, powered by Clever Cloud
+    </pre>
+
+    [see supported addons](https://www.clever-cloud.com/doc/addons/clever-cloud-addons/#available-add-ons)
+
+3. Setup env var `clever env set CC_PRE_RUN_HOOK "cp ./clevercloud/application-clevercloud.yml ./application-prod.yml"`
+
+4. Enable dedicated build `clever scale --build-flavor M`
+
+    [see dedicated build](https://www.clever-cloud.com/doc/admin-console/apps-management/#dedicated-build)
+
+
+## Configure your jhipster application
+1. Add a `clevercloud/` folder in base directory
+
+2. create `clevercloud/application-clevercloud.yml` for using predefined clever cloud addon env var
+
+    For PostgreSQL
+    <pre>
+    spring:
+        datasource:
+            type: com.zaxxer.hikari.HikariDataSource        
+            url: jdbc:postgresql://${POSTGRESQL_ADDON_HOST}:${POSTGRESQL_ADDON_PORT}/${POSTGRESQL_ADDON_DB}?useUnicode=true&characterEncoding=utf8&useSSL=false
+            username: ${POSTGRESQL_ADDON_USER}
+            password: ${POSTGRESQL_ADDON_PASSWORD}
+            hikari:
+                maximumPoolSize: 2
+    </pre>
+
+    For MySQL
+    <pre>
+    spring:
+        datasource:
+            type: com.zaxxer.hikari.HikariDataSource        
+            url: jdbc:mysql://${MYSQL_ADDON_HOST}:${MYSQL_ADDON_PORT}/${MYSQL_ADDON_DB}?useUnicode=true&characterEncoding=utf8&useSSL=false
+            username: ${MYSQL_ADDON_USER}
+            password: ${MYSQL_ADDON_PASSWORD}
+            hikari:
+                maximumPoolSize: 2
+    </pre>
+
+    For MongoDB
+    <pre>
+    spring:
+      data:
+        mongodb:
+          uri: ${MONGODB_ADDON_URI}
+          database: ${MONGODB_ADDON_DB}
+    </pre>
+
+
+
+3. add json file
```

Comment:
maybe add one sentence whats the purpose of this json file (like for the .yml file)

Reply:
> add a json file that contain the goal field to indicate how to start your application
is it better ?